samples/extensions/logic_op_dynamic_state/README.md

```diff
@@ -0,0 +1,166 @@
+<!--
+- Copyright (c) 2023, Mobica Limited
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
+# Logic operations dynamic state
+
+![Sample](./images/logic_op_dynamic_state_screenshot.png)
+
+## Overview
+
+The [VK_EXT_extended_dynamic_state2](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_extended_dynamic_state2.html) extension allows to use dynamic states e.g. the VK_DYNAMIC_STATE_LOGIC_OP_EXT. This may help an application to change a logical operation used without creating a new pipeline.
+
+The sample demonstrates usage of this extension with dynamically changed logical operations applied to blending.
+
+## Surface format setting
+
+Logical operations are applied only for [signed and unsigned integer and normalized integer framebuffers](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#VkLogicOp).
+
+In the sample surface format is changed to `VK_FORMAT_B8G8R8A8_UNORM` by overriding virtual method `create_render_context` (derived from `ApiVulkanSample` class).
+
+```C++
+auto surface_priority_list = std::vector<VkSurfaceFormatKHR>{
+    {VK_FORMAT_B8G8R8A8_UNORM, VK_COLORSPACE_SRGB_NONLINEAR_KHR},
+};
+render_context = platform.create_render_context(*device.get(), surface, surface_priority_list);
+```
+
+## Enabling logical operations in pipeline creation
+
+In the sample two pipelines are created: 
+
+- a pipeline for a background model (the background pipeline);
+- a pipeline for a cube model (the baseline pipeline). 
+
+The cube model is used to present blending effect done using the dynamic logical operations.
+
+`logicOpEnable` in `VkPipelineColorBlendStateCreateInfo` structure [controls if logical operations should be applied](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPipelineColorBlendStateCreateInfo.html).
+
+`logicOpEnable` is set to `VK_TRUE` during creation of the baseline pipeline in `create_pipeline` method.
```

Comment:
I think it should be:
(...) in the `create_pipeline` method.

Reply:
fixed